src/youtube.js

```diff
@@ -1127,7 +1133,23 @@ class Youtube extends FakeEventTarget implements IEngine {
     });
   }
 
+  _handleClipTo(): void {
+    if (this._clipToInterval) {
+      clearInterval(this._clipToInterval);
+    }
+    const { clipTo } = this._config.sources;
+    if (this._api && clipTo && this.currentTime < clipTo) {
+      this._clipToInterval = setInterval(() => {
```

Comment:
so the "youtube" way to do clipping is basically to set start and stop times of the video ?
what about the clipping feature in youtube ? it doesn't help us ?

Reply:
we can't use the clipping feature of youtube it's an in-house feature and is not exposed. after speaking with Product, the requirement is to create chapters based on seekFrom and clipTo configurations.